### DISCUSSION TITLE
Deploying a similar script to cswindiag on Windows PowerShell to Linux servers using RTR

### DISCUSSION THREAD

**Discussion by NSH531** [CrowdStrike/falconpy#1047] [Q&A]
Hi everyone,

I'm working on a project to push a similar script to cswindiag on Windows PowerShell bash script to Linux servers using RTR. I'm wondering if anyone has any experience with this or can point me in the right direction.

The script I'm using is a simple one that collects basic system information, such as the operating system version, installed software, and open ports. I'm using RTR to push the script to the Linux servers.

I've been able to get the script to run successfully on a few servers, but I'm having trouble getting it to run on all of them. I'm not sure if it's a permissions issue or something else.
I glad for any help!
tnx,

Netanel

**Comment by jshcodes** [comment]
Hi @NSH531 -

If the code is working on some endpoints, and not on others, it might not be an issue with the code that interacts with the API.  (i.e. potentially an error with the uploaded BASH script and the target endpoint.) 

Can we see an example of the error messages you're receiving on failure?

Are you getting different error messages for the failures depending on the endpoint, or is it the same failure message for all of them?

**Comment by NSH531** [comment]
Hi, 
I mean a command equivalent to cswindiag for testing a script that pushes in RTR to a group of Linux servers and I want to check something there that CSWINDIAG does not allow me.

Best regards
Netanel
